packages/apsara-ui/src/Combobox/Combobox.stories.tsx

```diff
@@ -35,3 +37,36 @@ MultiSelectWithSearch.args = {
     mode: "multiple",
     optionFilterProp: "label",
 };
+
+export const WithAsyncOptions = () => {
+    const [searchQuery, setSearchQuery] = useState<string>('');
+    const [isSearching, setIsSearching] = useState<boolean>(false);
+    const [opts, setOpts] = useState<SelectOptionType>();
+    
+    const search = (q: string) => {
+        setIsSearching(true);
```

Comment:
let's also set the searchQuery here, this will also fix the lint issue

Reply:
oh it is actually unused, forgot to remove it.